css/styles.css

```diff
@@ -497,10 +497,6 @@ section#developers .section-header {
 
```

Comment:
Could you create another PR for https://github.com/eclipse/smarthome/blob/master/docs/css/styles.css as well to keep it in sync?